### PR TITLE
Update metadata: author, ms.author and remove by lines

### PR DESCRIPTION
**Pull request by wadepickett**
Updating meta data for author and ms.author and removing by lines per doc manager request.

27 topics.

Note: Excludes gRPC topics which will be updated in different PR and there are 8 other topics on the learn/training platform/repo.

### CODE REVIEW

**Comment by wadepickett** [comment]
@mcleblanc FYI.  2 topics were updated for the metadata previously + this pr is 29 total.  gRPC topics will be changed in a separate PR.

File: aspnetcore/fundamentals/websockets.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 ---
 title: WebSockets support in ASP.NET Core
-author: rick-anderson
+author: wadepickett
 description: Learn how to get started with WebSockets in ASP.NET Core.
 monikerRange: '>= aspnetcore-3.1'
-ms.author: riande
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/websockets.md:6]: Why replace the MS author? That can act as the backup author.

Comment (wadepickett) [reply aspnetcore/fundamentals/websockets.md:6]: @Rick-Anderson, per doc manager, (as stated above).  I'm supposed to own these entirely going forward.  Both author and ms.author settings.

File: aspnetcore/tutorials/first-mongo-app.md [hunk 2]
```diff
@@ -1,17 +1,15 @@
 ---
 title: Create a web API with ASP.NET Core and MongoDB
-author: prkhandelwal
+author: wadepickett
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/first-mongo-app.md:3]: revert. WE need original author and expert

File: aspnetcore/tutorials/first-mongo-app.md [hunk 3]
```diff
@@ -1,17 +1,15 @@
...
-ms.author: scaddie
+ms.author: wpickett
 ms.custom: "mvc, seodec18"
 ms.date: 03/24/2022
 uid: tutorials/first-mongo-app
 ---
 # Create a web API with ASP.NET Core and MongoDB
 
-By [Pratik Khandelwal](https://twitter.com/K2Prk) and [Scott Addie](https://twitter.com/Scott_Addie)
-
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/first-mongo-app.md:14]: just delete scott so we can use Pratik

File: aspnetcore/tutorials/first-mvc-app/working-with-sql.md [hunk 4]
```diff
@@ -12,8 +12,6 @@ ms.custom: contperf-fy21q3
 
 :::moniker range=">= aspnetcore-7.0"
 
-By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Jon P Smith](https://twitter.com/thereformedprog).
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/first-mvc-app/working-with-sql.md:15]: Why remove Jon? He helps frequently on this an other EF Core problems

File: aspnetcore/tutorials/signalr-typescript-webpack.md [hunk 5]
```diff
@@ -3,7 +3,7 @@ title: "Tutorial: Get started with ASP.NET Core SignalR using TypeScript and Web
 author: ssougnez
 description: This tutorial provides a walkthrough of bundling and building an ASP.NET Core SignalR web app using TypeScript and Webpack.
 monikerRange: ">= aspnetcore-2.1"
-ms.author: bradyg
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/signalr-typescript-webpack.md:6]: revert.
don't delete expert authors

File: aspnetcore/tutorials/signalr.md [hunk 6]
```diff
@@ -2,7 +2,7 @@
 title: Get started with ASP.NET Core SignalR
 author: bradygaster
 description: In this tutorial, you create a chat app that uses ASP.NET Core SignalR.
-ms.author: bradyg
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/signalr.md:5]: revert

**Review by Rick-Anderson — CHANGES_REQUESTED** [review]

**Comment by Rick-Anderson** [comment]
> I'm supposed to own these entirely going forward. Both author and ms.author settings.

`ms.author:` provides a backup if  responsible author (`author:`) is unavailable.

**Comment by wadepickett** [comment]
@mcleblanc for further guidance since he asked for ms.author, author to be changed and by lines to be removed for these topics.

**Comment by Rick-Anderson** [comment]
> @mcleblanc for further guidance since he asked for ms.author, author to be changed and by lines to be removed for these topics.

I'm asking to keep customer bylines and expert MS author so we have the right contact for technical questions. OK to remove me because we know I'm the backup contact when you're not available or need technical help.

**Comment by wadepickett** [comment]
Understood, thanks. I will let the doc manager weigh in since it was his request.

**Review by Rick-Anderson — APPROVED** [review]

File: aspnetcore/tutorials/first-mongo-app.md (hunk 2 above)
Comment (Rick-Anderson) [inline aspnetcore/tutorials/first-mongo-app.md:3]:
```suggestion
author: wadepickett
<!-- author: prkhandelwal -->
```

**Comment by Rick-Anderson** [comment]
@wadepickett  per Mark
_That’s fine to keep the byline._

**Comment by wadepickett** [comment]
Keep byline, put experts in comments:
Sounds good.  Thanks @mcleblanc and @Rick-Anderson.

**Comment by Rick-Anderson** [comment]
By ~[Rick Anderson](https://twitter.com/RickAndMSFT) and~ [Jon P Smith](https://twitter.com/thereformedprog).

File: aspnetcore/tutorials/first-mvc-app/working-with-sql.md [hunk 7]
```diff
@@ -12,8 +12,6 @@ ms.custom: contperf-fy21q3
 
 :::moniker range=">= aspnetcore-7.0"
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/first-mvc-app/working-with-sql.md:13]:
```suggestion
:::moniker range=">= aspnetcore-7.0"
By [Jon P Smith](https://twitter.com/thereformedprog)
```

File: aspnetcore/tutorials/razor-pages/sql.md [hunk 8]
```diff
@@ -1,16 +1,14 @@
...
-author: rick-anderson
+author: wadepickett
 description: Part 4 of tutorial series on Razor Pages.
-ms.author: riande
+ms.author: wpickett
 ms.date: 10/24/2022
 ms.custom: contperf-fy21q2
 uid: tutorials/razor-pages/sql
 ---
 # Part 4 of tutorial series on Razor Pages
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/razor-pages/sql.md:11]: byline

File: aspnetcore/tutorials/razor-pages/sql.md [hunk 9]
```diff
@@ -1,16 +1,14 @@
...
 
-By [Rick Anderson](https://twitter.com/RickAndMSFT) and [Joe Audette](https://twitter.com/joeaudette)
-
 :::moniker range=">= aspnetcore-7.0"
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/razor-pages/sql.md:12]:
```suggestion
By  [Joe Audette](https://twitter.com/joeaudette)
:::moniker range=">= aspnetcore-7.0"
```

File: aspnetcore/tutorials/signalr-typescript-webpack.md [hunk 10]
```diff
@@ -3,7 +3,7 @@ title: "Tutorial: Get started with ASP.NET Core SignalR using TypeScript and Web
...
+ms.author: wpickett
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/signalr-typescript-webpack.md:6]:
```suggestion
ms.author: wpickett
<!--ms.author: bradyg-->
```

File: aspnetcore/tutorials/signalr.md [hunk 11]
```diff
@@ -2,7 +2,7 @@
...
+ms.author: wpickett
```
Comment (Rick-Anderson) [inline aspnetcore/tutorials/signalr.md:5]:
```suggestion
ms.author: wpickett
<!--ms.author: bradyg-->
```

Comment (wadepickett) [reply aspnetcore/tutorials/signalr.md:6]: @Rick-Anderson,  this is in draft, give me few minutes.  I will re-ask for a review when it is ready.

**Comment by wadepickett** [comment]
@Rick-Anderson, sorry, we were both changing content at the same time.  Ready for review now.  I put the by lines back in that also included you, but after your last edit here I see you removed yourself but added the dev team author by line.  Is this now the way it should be or do you want your own by line removed?  The instructions just said add by line back in.  It wasn't very specific.

**Comment by wadepickett** [comment]
Doc manager approved line changes since original review, merging.

**Pull request merged**